src/components/text/index.tsx

```diff
@@ -39,6 +39,7 @@ export type TextProps = RNTextProps & TypographyModifiers & ColorsModifiers & Ma
    */
   animated?: boolean;
   textAlign?: string;
+  style?: StyleProp<TextStyle | Animated.AnimatedProps<TextStyle>>;
```

Comment:
You need to `Omit<RNTextProps, 'style'>` in `TextProps`

Reply:
Yes, weirdly it didn't throw an error on that, anyway fixed it.

Reply:
We should move more screens to tsx (found it there)